packages/radix-ui-themes/src/components/callout.tsx

```diff
@@ -74,13 +66,11 @@ type CalloutTextElement = React.ElementRef<'p'>;
 type CalloutTextProps = ComponentPropsAs<typeof Text, 'p'>;
 const CalloutText = React.forwardRef<CalloutTextElement, CalloutTextProps>(
   ({ className, ...props }, forwardedRef) => {
-    const { color, size, highContrast } = React.useContext(CalloutContext);
+    const { size } = React.useContext(CalloutContext);
     return (
       <Text
         as="p"
         size={mapResponsiveProp(size, mapCalloutSizeToTextSize)}
-        color={color}
-        highContrast={highContrast}
```

Comment:
Was unnecessary

Reply:
- How is it unnecessary then? Where is color/high-contrast set then?
- If unnecessary, should we update the context definition?

Reply:
The `data-accent-color` attribute and text color CSS is set on Callout Root anyway. See the data-accent-color repetition here (this is current):

<img width="776" alt="image" src="https://github.com/radix-ui/themes/assets/8441036/e6ebc293-89a4-4bac-a397-538ff1dad58c">

Reply:
Updated the context 950bea0b9266c0fb237f4a4d3226ba0037d0ebf0

Reply:
Yeah good point, not sure why I did that in the first place, seems odd.